crates/libs/windows/src/core/factory_cache.rs

```diff
@@ -118,6 +128,18 @@ pub fn factory<C: RuntimeName, I: Interface>() -> Result<I> {
     }
 }
 
+unsafe fn get_activation_factory(library: &[u8], name: &HSTRING) -> Result<*mut std::ffi::c_void> {
```

Comment:
This can just be merged into the `factory_from_library` function suggested above.

Reply:
This is used by both `factory` and `factory_from_library` so left it in place.